public/content/documentation/web/page-level/form.md

```diff
@@ -1,80 +1,598 @@
 ## General Notes
 
-How to test a form
+How to test a web form for accessibility.
 
 ## Condensed
 
 ### #a11y - Web Accessibility Acceptance Criteria
 
-How to test a header
+How to test a web form for accessibility
 
 1. Test keyboard only, then screen reader + keyboard actions
-
-   - Skip-links: Focus moves directly to the header or navigation
-
-   - Tab: Nothing, headings are not focusable unless they are actionable
-
-   - Arrow-keys: Headings are browsed
+   - Skip-links: Not commonly used for form entry, but useful for jumping to a form landmark.
+   - Tab: Focus moves to each form control in logical order, strong visual focus indicator.
+   - Forms mode: Enabled when screen reader lands on a form field.
 
 2. Test mobile screenreader gestures
-
-   - Swipe: Focus moves directly to the header or navigation
-
-   - Doubletap: This typically activates most elements
+   - Swipe: Focus moves to form controls inside the form.
+   - Singletap: Selects a form control.
+   - Doubletap: Activates controls as expected.
+   - Forms mode: Automatically enabled.
 
 3. Listen to screenreader output on all devices
+   - Role: Each input and group is correctly identified (e.g. textbox, combobox, radiogroup).
```

Comment:
Role should still be about the role of the form itself: It is discoverable with screenreader shortcuts as a form landmark along with its name